Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+import groovy.json.JsonOutput
+
+/** Tox environment */
+def environment = 'e2e'
+
+/** Run Tox
+ *
+ * @param environment test environment to run
+*/
+def runTox(environment) {
+  def options = env.PYTEST_ADDOPTS ?: ''
```

Comment:
The env is already passed to sh, so you don't need to set them in groovy as well.